src/static/mermaid.mjs

```diff
@@ -54,8 +54,8 @@ async function render(definition, config, bgColor, size) {
     }
   } catch (error) {
     console.error('Failed to render', error);
-    if (isSyntaxErrorFromMermaid(error.toString())) {
-      throw new Error('Syntax error in graph');
+    if (isUnknownDiagramError(error.toString())) {
+      throw new Error('Unknown diagram error');
     }
     throw error;
```

Comment:
**suggestion:** I'd prefer not to include error's call stack in the response. For example:

```sh
# Let's exclude the following information
o1.parseError (file:///.../node_modules/mermaid/dist/chunks/mermaid.esm.min/flowDiagram-FKN2Y2MS.mjs:9:31696)
```

```suggestion
    throw error.message;
```